examples/js/loaders/GLTF2Loader.js

```diff
@@ -1642,11 +1668,16 @@ THREE.GLTF2Loader = ( function () {
 
 					materialType = DeferredShaderMaterial;
 
-					var technique = json.techniques[ material.technique ];
+					var extension = extensions[ EXTENSIONS.KHR_TECHNIQUE_WEBGL ];
```

Comment:
Similar to spec-gloss, could we do something like this to factor most of this code out of this large `loadMaterials` method?

```js

materialType = DeferredShaderMaterial;
extensions[ EXTENSIONS.KHR_TECHNIQUE_WEBGL ].extendParams( materialParams, material, dependencies );

```